### PR TITLE
Doc 10052

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

**Review by ThreeJackSparrow — APPROVED** [review]

File: modules/release-notes/pages/relnotes.adoc [hunk 1]
```diff
@@ -5,6 +5,196 @@
...
+== Known Issues
+
+=== Install and Upgrade
+
+[#table_fixedissues_v704-installandupgrade,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52313[MB-52313^]
+| *Summary*: Analytics Service crashes during upgrade
```
Comment (b33f) [inline modules/release-notes/pages/relnotes.adoc:194]: We should mention this is only on Windows.  And there's a workaround per the MB comments.

File: modules/release-notes/pages/relnotes.adoc [hunk 2]
```diff
@@ -5,6 +5,196 @@
...
+|===
+
+=== Analytics Service
+
+[#table_fixedissues_v704-analyticsservice,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-51676[MB-51676^]
+| *Summary*: To investigate the crash in MB-49471
```
Comment (b33f) [inline modules/release-notes/pages/relnotes.adoc:130]: This isn't useful to customers, needs a better description from engineering.  Also this is Windows only.

File: modules/release-notes/pages/relnotes.adoc [hunk 3]
```diff
@@ -5,6 +5,196 @@
...
+| *Summary*: Data is not completely ingested into datasets
+
+| https://issues.couchbase.com/browse/MB-51482[MB-51482^]
+| *Summary*: Node is not upgraded and stuck trying to register with CC
+
+| https://issues.couchbase.com/browse/MB-51488[MB-51488^]
+| *Summary*: Ignore byte order marks when parsing external data
+
+| https://issues.couchbase.com/browse/MB-51700[MB-51700^]
+| *Summary*: CBAS service keeps crashing when one of the CBAS nodes is failed over
```
Comment (b33f) [inline modules/release-notes/pages/relnotes.adoc:142]: It's called the Analytics Service in all our documentation and UI etc.

File: modules/release-notes/pages/relnotes.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc:130]: Shall I remove?

File: modules/release-notes/pages/relnotes.adoc (hunk 3 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc:142]: ok

File: modules/release-notes/pages/relnotes.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc:194]: We have to follow up with the workaround, I understand. I can add that this is windows.

File: modules/release-notes/pages/relnotes.adoc (hunk 2 above)
Comment (westmann) [reply modules/release-notes/pages/relnotes.adoc:130]: I think that we should not add a release note for MB-51676.

File: modules/release-notes/pages/relnotes.adoc [hunk 4]
```diff
@@ -5,6 +5,106 @@
...
+
+[#table_fixedissues_v704-analyticsservice,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-51676[MB-51676^]
+| *Summary*: To investigate the crash in MB-49471
+
+| https://issues.couchbase.com/browse/MB-51483[MB-51483^]
+| *Summary*: Data is not completely ingested into datasets
```
Comment (westmann) [inline modules/release-notes/pages/relnotes.adoc:133]: This was an internal test with no customer impact, I think that we should not add a release note.

File: modules/release-notes/pages/relnotes.adoc [hunk 5]
```diff
@@ -5,6 +5,106 @@
...
+| *Summary*: Node is not upgraded and stuck trying to register with CC
+
+| https://issues.couchbase.com/browse/MB-51488[MB-51488^]
+| *Summary*: Ignore byte order marks when parsing external data
+
+| https://issues.couchbase.com/browse/MB-51700[MB-51700^]
+| *Summary*: CBAS service keeps crashing when one of the CBAS nodes is failed over
+
+| https://issues.couchbase.com/browse/MB-51412[MB-51412^]
+| *Summary*:  Graceful failover + delta recovery + rebalance fails
```
Comment (westmann) [inline modules/release-notes/pages/relnotes.adoc:145]: This was fixed in 6.6.5 and forward ported to 7.0.4. We didn't mention it in the 6.6.5 release notes and we should not add it now.

File: modules/release-notes/pages/relnotes.adoc (hunk 4 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc:133]: ok

File: modules/release-notes/pages/relnotes.adoc (hunk 5 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc:145]: ok

File: modules/release-notes/pages/relnotes.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc:130]: Removed.

File: modules/release-notes/pages/relnotes.adoc [hunk 6]
```diff
@@ -5,6 +5,106 @@
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
+[#release-704]
+== Release 7.0.4 (May 2022)
+Couchbase Server 7.0.4 was released in May 2022.
+This maintenance release contains fixes to issues.
+
+[#fixed-issues-704]
```
Comment (westmann) [inline modules/release-notes/pages/relnotes.adoc:13]: Do we also have a section for known issues (e.g. https://issues.couchbase.com/browse/MB-51699)?

Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc:13]: There is a section for Known Issues at the end of the 7.0.4 section. It contains only MB-52313. Do you wish to add MB-51699?

File: modules/release-notes/pages/relnotes.adoc [hunk 7]
```diff
@@ -5,6 +5,106 @@
...
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-51676[MB-51676^]
+| *Summary*: To investigate the crash in MB-49471
+
+| https://issues.couchbase.com/browse/MB-51483[MB-51483^]
+| *Summary*: Data is not completely ingested into datasets
+
+| https://issues.couchbase.com/browse/MB-51482[MB-51482^]
+| *Summary*: Node is not upgraded and stuck trying to register with CC
```
Comment (westmann) [inline modules/release-notes/pages/relnotes.adoc:136]: Do we add release notes for backports? (MB-51482 and MB-51488)?

Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc:136]: I'm open to your guidance on this.

Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc:136]: If  issues are included in the filter output, then they are considered candidates. That is the process.

File: modules/release-notes/pages/relnotes.adoc (hunk 6 above)
Comment (westmann) [reply modules/release-notes/pages/relnotes.adoc:13]: Yes, please add it. We've also provided a summary and the workaround on the ticket.

**Pull request merged**